stable/artifactory/templates/artifactory-serviceaccount.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-{{- if .Values.serviceAccount.annotations }}
+{{- with .Values.serviceAccount.annotations }}
```

Comment:
@eblaas  Thanks for contribution ! can you please do the same changes in artifactory-ha chart in same PR